selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/RevertCommitTest.java

```diff
@@ -72,10 +72,10 @@
 
   @BeforeClass
   public void prepare() throws Exception {
-    URL resource = getClass().getResource("/projects/git-pull-test");
     testUserPreferencesServiceClient.addGitCommitter(gitHubUsername, productUser.getEmail());
+    URL resource = getClass().getResource("/projects/git-pull-test");
     testProjectServiceClient.importProject(
-        ws.getId(), Paths.get(resource.toURI()), PROJECT_NAME, ProjectTemplates.MAVEN_SPRING);
+        ws.getId(), Paths.get(resource.toURI()), PROJECT_NAME, ProjectTemplates.NODE_JS);
```

Comment:
Again, why is this necessary?

Reply:
It isn't a maven project. 
If we keep maven template then jdt.ls seems continuously import the project.